docs/reference/manifest/extendedpermissions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ExtendedPermissions element in the manifest file
+description: ''
+ms.date: 03/04/2020
+localization_priority: Normal
+---
+
+# ExtendedPermissions element
+
+Defines the extended permissions the add-in needs to work. The `ExtendedPermissions` element is a child element of [VersionOverrides](versionoverrides.md).
+
+> [!IMPORTANT]
+> This element is only available in the [Outlook add-ins preview requirement set](../objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) against Exchange Online. Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.
+
+## Child elements
+
+|  Element |  Required  |  Description  |
+|:-----|:-----|:-----|
+|  [ExtendedPermission](extendedpermission.md)    |  Yes   | Defines the extended permission needed for the add-in to work. At least one entry should be included in the `ExtendedPermissions` collection. |
```

Comment:
This is not required. It is optional but required if they want to use a feature that needs this.

Reply:
Ok - so technically someone can have an empty ExtendedPermissions node?